### DISCUSSION TITLE
Sigmastar SDK and developer tools and toolchain

### DISCUSSION THREAD

**Discussion by Hello-dot-Jpg** [nk64/GT4286#14] [Q&A]
English Developer information about "FlyThings" development 
https://developer.flythings.cn/en/

SDK and toolchain download

https://developer.flythings.cn/en/

**Comment by Hello-dot-Jpg** [comment]
opps screwed up the toolchain link

https://developer.flythings.cn/zh-hans/download.html

**Reply by Hello-dot-Jpg** [reply]
to make it even easier here is a direct link to the correct toolchain

https://download.flythings.cn/toolchain/ssd.tar.gz

**Reply by nk64** [reply]
Wow, the Chinese download page has a much more recent version of the IDE than the English download page.

**Comment by horsiezoo** [comment]
Has anyone managed to get the Flythings ide to compile anything? Extracted a ssd sdk.7z to …/sdk but the the ide keeps shouting at me in Chinese about not finding a sdk installed and not being able to find make.exe (it’s in the sdk). The documentation is not very clear as to the expected path so I guessed and copied it to multiple folders. Interesting when you create a project you see the origin of some of the various files on the sd card, eg the baudrate settings etc. This is obviously designed for a dev board that has full access to serial without needing to drill holes etc.(no insult there, that was a pretty good idea) 
I suppose an image could plausibly be compiled and hardware flashed onto the flash chip but there’s a brick in the making and not recommended.

**Reply by nk64** [reply]
I have not tried yet, need a good couple of hours to set aside for it.
